driver/configuration/provider_viper.go

```diff
@@ -62,6 +62,7 @@ const (
 	ViperKeySelfServiceLifespanLoginRequest        = "selfservice.login.request_lifespan"
 	ViperKeySelfServiceLogoutRedirectURL           = "selfservice.logout.redirect_to"
 	ViperKeySelfServiceLifespanProfileRequest      = "selfservice.profile.request_lifespan"
+	ViperKeySelfServiceTimeoutPrivileged           = "selfservice.profile.privileged_timeout"
```

Comment:
We are calling everything else „lifespan“ so let’s stick to one convention. How about:

- privileged_lifespan

?

Reply:
I was not convinced by the naming at this point anyway but is it really a lifespan? The session is generally still valid but not new enough to do everything. What about "privileged_authentication_after"

Reply:
what about `privileged_session_max_age`?